intg-androidtv/setup_flow.py

```diff
@@ -44,6 +44,8 @@ class SetupSteps(IntEnum):
 _cfg_add_device: bool = False
 _discovered_android_tvs: list[dict[str, str]] = []
 _pairing_android_tv: tv.AndroidTv | None = None
+_use_chromecast: bool = True
```

Comment:
Please change default to `False`:
Until we have this feature better tested and are sure that reconnection works, I'd like to release it as a preview feature.